Source/AwsCommonRuntimeKit/http/CRTHttpResponse.swift

```diff
@@ -4,9 +4,6 @@ import AwsCHttp
 import AwsCIo
 
 public final class CRTHttpResponse: HttpMessage {
```

Comment:
We can probably just remove this class. We can get the response status code from HttpStream class.